operator/pkg/ciliumendpointslice/manager.go

```diff
@@ -441,7 +459,27 @@ func (c *cesMgr) getCEPCountInCES(cesName string) (cnt int) {
 
 // Returns the total count of CESs in local cache
 func (c *cesMgr) getCESCount() int {
-	return (c.desiredCESs.getCESCount())
+	return c.desiredCESs.getCESCount()
+}
+
+func (c *cesMgr) getAllCESs() []cesOperations {
+	var cess []cesOperations
+	for _, ces := range c.desiredCESs.getAllCESs() {
```

Comment:
Please consider preallocating `cess` or using `copy` (I am not sure of the type)

Reply:
Preallocated the slice